.env

```diff
@@ -0,0 +1,9 @@
+CLIO_MANAGE_CLIENT_ID=YOUR_APP_CLIENT_ID
```

Comment:
Do we want to include the text about keeping your secrets secret in this file since the local_en.yml file is removed. 
https://github.com/clio/example-third-party-application/pull/7/files#diff-f81833d8f67218aa5c20e6b33137a7d509101de4d40186030e9f78b744ff139bL1

Reply:
Added them back!